sycl/test-e2e/bindless_images/sampling_1D_USM_host.cpp

```diff
@@ -0,0 +1,126 @@
+// REQUIRES: aspect-ext_oneapi_bindless_images
+// REQUIRES: aspect-ext_oneapi_bindless_images_sample_1d_usm
+
+// UNSUPPORTED: hip
+// UNSUPPORTED-INTENDED: Host USM backed image support is not yet enabled in UR
+// adapter. Also, when provionally enabled, the test crashes upon image
+// creation, whereas Device USM backed images do not crash. This issue is
```

Comment:
What do you mean under "this issue is undetermined"? Do you suspect an issue in HIP backend? HIP adapter? Should we probably report it?

Reply:
The source of the problem is not yet known, but we are tracking the issue internally.